src/components/common/Tabs/RectangleProgressTabs/styled.tsx

```diff
@@ -1,15 +1,15 @@
 import { styled } from '@mui/material';
 import { DefaultThemeType } from '../../../../styles/theme/types';
 
-export type TabColorType = 'secondary' | 'error' | 'hidden';
+export type TabColorType = 'default' | 'active' | 'right' | 'error' | string;
 const tabColors = (
```

Comment:
Function should return ready result. Not an object, that should be used to find result, but result

Reply:
fixed